### PR TITLE
Feature/help

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceLatencyProbe.java [hunk 1]
```diff
@@ -27,14 +24,15 @@
 @Component
 public class FogbowServiceLatencyProbe extends Probe {
 
-    private static final String PROBE_LABEL = "service_latency_probe";
+    private static final String PROBE_NAME = "service_latency";
     private static final Logger LOGGER = LogManager.getLogger(FogbowServiceLatencyProbe.class);
     private static final String COMPUTE_JSON_KEY = "COMPUTE";
     private static final String NETWORK_JSON_KEY = "NETWORK";
     private static final String VOLUME_JSON_KEY = "VOLUME";
+    private static final String HELP = "Latency is measured by the time that elapses between the order being opened until it is available.";
```
Comment (joivo) [inline probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceLatencyProbe.java]: Replace by something as `Latency until orders are available.`

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceSuccessRateProbe.java [hunk 2]
```diff
@@ -16,19 +16,22 @@
...
+ * FogbowServiceSuccessRateProbe is responsible for measuring the success rate in requesting a
+ * resource. The success rate is measured by the relationship between the number of orders that
+ * failed after requests ({@link OrderState#FAILED_AFTER_SUCCESSFUL_REQUEST}) and the number of
+ * orders opened ({@link OrderState#OPEN}).
  */
 @Component
 public class FogbowServiceSuccessRateProbe extends Probe {
 
-    private static final String PROBE_LABEL = "service_success_rate";
+    private static final String PROBE_NAME = "service_success";
```
Comment (joivo) [inline probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceSuccessRateProbe.java]: Keep the `service_success_rate`. It's more descriptive.

**Review by joivo — CHANGES_REQUESTED** [review]

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceLatencyProbe.java (hunk 1 above)
Comment (wesleymonte) [reply probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceLatencyProbe.java]: Done.

File: probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceSuccessRateProbe.java (hunk 2 above)
Comment (wesleymonte) [reply probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceSuccessRateProbe.java]: Done.

**Review by joivo — APPROVED** [review]

**Pull request merged**